providers/os/resources/os.lr

```diff
@@ -929,6 +931,28 @@ iptables.entry {
   chain string
 }
 
+fstab @defaults("path") {
+  init(path? string)
+  path string
+
+  entries() []fstab.entry
+}
+
+private fstab.entry @defaults("device mountpoint") {
+  // Device referenced in the fstab, e.g. LABEL=rootfs
+  device string
+  // Mountpoint, e.g. '/'
+  mountpoint string
+  // Filesystem type, e.g. ext4
```

Comment:
```suggestion
  // File system type, e.g. ext4
```
I try to follow the Google dev style guide as much as we can. https://developers.google.com/style/word-list#file-system